src/controllers/document_upload/document.upload.controller.ts

```diff
@@ -157,10 +159,10 @@ const getUploadFinishedCallback = (req: Request,
 
       // TODO OBJ-101 - file type error - uncomment below should do it
       // // render errors in the view
-      // if (e.status === 415) {
-      //   return await
-      //     displayError(req, res, UploadErrorMessages.INVALID_MIME_TYPES, uploadResponderStrategy, attachments);
-      // }
+      if (e.status === 415) {
```

Comment:
415 is a magic number! Would be good to find out what this error code actually means, define a well-named constant for it and use that.

Reply:
Not sure what you mean by magic number - but will extract a constant if there are no references to this.

Reply:
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

And thanks.

Reply:
Updated this.